cmd/ndm_daemonset/probe/eventhandler.go

```diff
@@ -109,34 +109,18 @@ func (pe *ProbeEvent) deleteBlockDeviceEvent(msg controller.EventMessage) {
 	isGPTBasedUUIDEnabled := features.FeatureGates.IsEnabled(features.GPTBasedUUID)
 
 	for _, device := range msg.Devices {
-		// create the new UUID for removing the device
+		// create the new UUID and legacy UUID for removing the device
```

Comment:
Can we expand on this comment a little more. Unable to understand the reason for refactoring by just looking the changes in this file.

Reply:
Done. Code has been refactored and moved out.